resource/README.md

```diff
@@ -0,0 +1,10 @@
+# OpenCensus Library Resource Package
+This documentation serves to document the "look and feel" of the open source resource package.
+It describes they key types and overall behavior.
+
+The primary purpose of resources as a first-class concept in the core library is decoupling
```

Comment:
It would be useful to start with a definition of what a resource is.

Reply:
Please resolve this. I think it is important to specify what a "resource" is, maybe include examples.